### PR TITLE
<ListItemSelect/> - fix type

### PR DESCRIPTION
**Pull request by EgoziE**

### CODE REVIEW

**Review by TalZa1810 — APPROVED** [review]

File: packages/wix-style-react/src/ListItemSelect/test/ListItemSelect.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ function listItemSelectBuilderWithAllProps() {
   const { disabled, id, overrideOptionStyle, value } = listItemSelectBuilder({
```
Comment (TalZa1810) [inline packages/wix-style-react/src/ListItemSelect/test/ListItemSelect.tsx:26]: label should be added here too

**Review by TalZa1810 — CHANGES_REQUESTED** [review]

File: packages/wix-style-react/src/ListItemSelect/test/ListItemSelect.tsx (hunk 1 above)
Comment (EgoziE) [reply packages/wix-style-react/src/ListItemSelect/test/ListItemSelect.tsx]: right

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6778/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6778

**Pull request merged**